### PR TITLE
Add readme, diagram and example rule config for gasket integration

### PR DESCRIPTION
**Pull request by Bairdo**

### CODE REVIEW

**Comment by Bairdo** [comment]
I will add all required faucet & gasket config files to 'gasket/'

e.g.
gasket/faucet/faucet.yaml
gasket/faucet/gasket/base-link022-acls.yaml
...

File: demo/README.link022.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
+# Gasket & [Link022](https://github.com/google/link022)
+
+## Getting Started
+First *read* over [Gasket's readme](https://github.com/bairdo/gasket/blob.master/docs/README.authentication.md) for a general overview of Gasket's authentication, and the [link022 readme](./Readme.md).
```
Comment (boleifu) [inline demo/README.link022.md]: Seems this (https://github.com/bairdo/gasket/blob.master/docs/README.authentication.md) links to a NOT FOUND PAGE.

This (./Readme.md) also does not work.

**Comment by Bairdo** [comment]
looks like most of the links are broken

**Comment by Bairdo** [comment]
All links working now.

File: demo/gasket/faucet/gasket/auth.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+dps:
+    faucet-1:
+        interfaces:
+            1:  
+                auth_mode: access
+
+hostapd:
+    host: 192.168.11.6
+    port: 8888
+    # in hostapd.conf 'ctl_interface...:udp:20000' might be how you specifiy what to do
```
Comment (zhiweicen) [inline demo/gasket/faucet/gasket/auth.yaml]: Change
in hostapd.conf 'ctl_interface...:udp:20000' might be how you specifiy what to do
to
in hostapd.conf 'ctl_interface...:udp:8888' might be how you specifiy what to do
to be consistent with the config ?

**Pull request merged**